omnibus/Gemfile.lock

```diff
@@ -79,20 +80,21 @@ GEM
     buff-shell_out (0.2.0)
       buff-ruby_engine (~> 0.1.0)
     builder (3.2.3)
-    byebug (9.0.6)
+    byebug (9.1.0)
     celluloid (0.16.0)
       timers (~> 4.0.0)
     celluloid-io (0.16.2)
       celluloid (>= 0.16.0)
       nio4r (>= 1.1.0)
-    chef-config (13.2.20)
+    chef-config (13.4.19)
       addressable
       fuzzyurl
       mixlib-config (~> 2.0)
       mixlib-shellout (~> 2.0)
     chef-sugar (3.5.0)
+    citrus (3.0.2)
```

Comment:
Is this inclusion going to require any changes related to license_scout?

https://github.com/mjackson/citrus#license

Reply:
we don't ship these gems, just use them to run omnibus, so no, it shouldn't

Reply:
ironically it's license scout that brings those in.